src/lib/queue_worker.js

```diff
@@ -22,26 +22,27 @@ function QueueWorker(tasksRef, processId, sanitize, processingFunction) {
       error;
   if (_.isUndefined(tasksRef)) {
     error = 'No tasks reference provided.';
-    logger.error('QueueWorker(): ' + error);
+    logger.debug('QueueWorker(): ' + error);
```

Comment:
I also switched to `winston.debug` throughout as all the errors can be handled by the calling code, and they can always be turned on by adjusting the winston log level